api/src/services/import-services/measurement/utils/measurement-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { CSVConfigUtils } from '../../../../utils/csv-utils/csv-config-utils';
+import { CSVParams } from '../../../../utils/csv-utils/csv-config-validation.interface';
+import { ICritterDetailed } from '../../../critterbase-service';
+import { MeasurementCSVStaticHeader } from '../import-measurements-service';
+
+/**
+ * Get the measurement row TSN getter - ie: Get the ITIS TSN from the row
+ *
+ * @param {Map<string, ICritterDetailed>} surveyAliasMap The survey alias map
+ * @param {CSVConfigUtils} utils The CSV config utils
+ * @returns {*} {(params: CSVParams) => number} The get measurement row TSN callback
+ */
+export const getTsnFromMeasurementRow = (
+  surveyAliasMap: Map<string, ICritterDetailed>,
```

Comment:
super nit: The name of this param might be more clear as something like "critterAliasMap" or "surveyCritterAliasMap"

Same for the calling function, which also names the variable "surveyAliasMap".

Reply:
Yea that's totally fair. I just checked and there's about 90 references, I'll probably just follow this PR with one that find and replaces those references.

Reply:
Yeah fair. Not actually a big deal, just caught my eye and I was thinking "surveys have aliases?"

Reply:
Wrote a note and will resolve in the next PR